app/src/main/java/org/mozilla/focus/fragment/InfoFragment.java

```diff
@@ -5,26 +5,25 @@
 
 package org.mozilla.focus.fragment;
 
-import android.net.Uri;
 import android.os.Bundle;
+import android.os.Message;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-import android.webkit.GeolocationPermissions;
-import android.webkit.ValueCallback;
-import android.webkit.WebChromeClient;
-import android.webkit.WebView;
 import android.widget.ProgressBar;
 
 import org.mozilla.focus.R;
+import org.mozilla.focus.tabs.TabChromeClient;
+import org.mozilla.focus.tabs.TabView;
+import org.mozilla.focus.tabs.TabViewClient;
 import org.mozilla.focus.utils.IntentUtils;
-import org.mozilla.focus.web.Download;
-import org.mozilla.focus.web.IWebView;
+import org.mozilla.focus.web.WebViewProvider;
 
 public class InfoFragment extends WebFragment {
     private ProgressBar progressView;
+    private ViewGroup webViewSlot;
```

Comment:
InfoFragment is extending WebFragment but it is not reusing WebFragment.tabView and WebFragment.webViewSlot?